CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.26.0] - 2022-01-14
```

Comment:
The order of `[Unreleased]` header and `[2.26.0] - 2022-01-14` header seems not right. Could you explain it a little bit?

Reply:
Oh good catch. Think Michael also pointed that out but I did not understand. Removed now.